src/fides/cli/utils.py

```diff
@@ -85,6 +87,15 @@ def check_server(cli_version: str, server_url: str, quiet: bool = False) -> None
         # The server is ratelimiting us
         echo_red("Server ratelimit reached. Please wait one minute and try again.")
         raise SystemExit(1)
+    elif health_response.status_code != 200:
+        echo_red(f"Server response: {health_response.text}")
```

Comment:
nit: might be nice to also include the status code here?

Reply:
Good call, added here: https://github.com/ethyca/fides/pull/5253/commits/222832d894e899ef01a6b25fd09dcae554fc5cfa